R/find_bad_apples.R

```diff
@@ -2,23 +2,98 @@
 #'
 #' @description
 #' This function uses a univariate approach to outlier detection.
-#' For each column with outliers (values that are 3 or more standard deviations from the mean),
+#' For each column with outliers (values that are 2 or more standard deviations from the mean),
 #' this function will create a reference list of row indices with outliers, and
 #' the total number of outliers in that column.
 #'
 #' Note: This function works best for small datasets with unimodal variable distributions.
 #'
 #' @param df A dataframe containing numeric data
 #'
-#' @return A dataframe with columns for 'Variable' (dataframe column name),
-#' 'Indices' (list of row indices with outliers),
-#' 'Total Outliers' (number of outliers in the column)
+#' @return A dataframe with columns for 'variable' (dataframe column name),
+#' 'total_outliers' (number of outliers in the column), and
+#' 'indices' (list of row indices with outliers)
 #'
 #' @examples
 #' find_bad_apples(df)
 #'
 #' @export
 
 find_bad_apples <- function(df) {
-  # insert code here
+
+    # Returns a warning and stops the function if the dataframe includes non-numeric values
```

Comment:
Same comment as in python package. Your comments are just describing what the code does. I don't think they are required since your code is well written :)

Reply:
Thanks, @SamEdwardes ! :)